src/sniffr.js

```diff
@@ -101,7 +102,8 @@
 
   Sniffr.prototype.sniff = function(userAgentString) {
     var self = this;
-    var userAgent = (userAgentString || navigator.userAgent || "").toLowerCase();
+    var fallbackUserAgent = isBrowser ? navigator.userAgent : ""
```

Comment:
A very minor comment, just to follow the same style as the rest of the file we might want to add a semicolon `;` at the end of line. Will fix this in a separate commit.

Reply:
Yeah, i tried to keep to coding style but that one escaped. Thanks for fixing